setup.cfg

```diff
@@ -61,6 +61,8 @@ test =
 docs =
     sphinx-rtd-theme
     sphinx-astropy
+roman =
+    roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git
```

Comment:
This is temporary, right? I don't think we can release to PyPI with a pin like this.

Reply:
Correct.